src/auditor/checks.py

```diff
@@ -42,12 +45,12 @@ def tag_case(tag, uppercased, articles):
 
 
 def get_group_from_table(metatable_dict_entry):
-    '''
+    """
     Return the appropriate group title based on either the SGID table name or
     the shelved category.
-    '''
+    """
 
-    SGID_name, _, item_category, _ = metatable_dict_entry
+    SGID_name, _, item_category, _ = metatable_dict_entry  # pylint: disable=invalid-name
```

Comment:
**thought**: i would prefer to use a "valid" python variable name like `sgid_name` and not have to disable a lint warning rather than properly case the acronym and have to disable a lint error.